src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -124,7 +124,13 @@ namespace {
     }
 
 }
+    void Schedule::handleAQUCT(HandlerContext& handlerContext) {
+        throw std::logic_error(OpmInputError::format("{keyword} in {file} line {line} is not supported as SCHEDULE keyword yet", handlerContext.keyword.location()));
+    }
 
+    void Schedule::handleAQUFETP(HandlerContext& handlerContext) {
+        throw std::logic_error(OpmInputError::format("{keyword} in {file} line {line} is not supported as SCHEDULE keyword yet", handlerContext.keyword.location()));
+    }
```

Comment:
Is there a reason for using `logic_error` here instead of [`OpmInputError`](https://github.com/OPM/opm-common/blob/093ead59464755c02349aae711cdba59a37367d4/opm/common/utility/OpmInputError.hpp#L49)?  The latter is explicitly intended for these kinds of situations.

Reply:
Thanks, there is no good reason but not thinking about it.